grpc4bmi/bmi_client_docker.py

```diff
@@ -58,6 +76,15 @@ def __init__(self, image, image_port=50051, host=None,
                                                remove=remove,
                                                detach=True)
         time.sleep(delay)
+        if not remove:
+            # Only able to reload, read logs when container is not in auto remove mode
+            self.container.reload()
+            if self.container.status == 'exited':
+                exitcode = self.container.attrs["State"]["ExitCode"]
+                logs = self.container.logs()
+                msg = f'Failed to start Docker container with image {image}'
```

Comment:
Should we also include the docker log in the error message?
```suggestion
                msg = f'Failed to start Docker container with image {image}. Container log: {logs}'
```

Also, would it make sense to point explicitly to a potential source of the error here? For #50 the following would have been helpful:
```suggestion
                msg = f'Failed to start Docker container with image {image}. This can happen e.g. if a nonexistent input_dir is specified.'
```

Reply:
I was unsure whether to add the log, as it might be long. Will add the log to the message.  
Let's add a check for nonexistent input dir.